client/index.html

```diff
@@ -1,27 +1,81 @@
 <!DOCTYPE html>
 <html lang="ko">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>배민 로그인</title>
-    <link rel="stylesheet" href="./assets/css/index.css">
-    <link rel="stylesheet" href="./assets/css/login.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>배민 로그인</title>
+  <link rel="stylesheet" href="./assets/css/index.css">
+  <link rel="stylesheet" href="./assets/css/main.css">
 </head>
 <body>
-    <header class="header-layout">
-        <button class="close-btn">
-            <img src="./assets/icons/x.svg" alt="close-btn"/>
-        </button>
-    </header>
-    <main class="login-page-layout">
-        <form class="login-form">
-            <input class="woowa-input login-form__id" placeholder="아이디를 입력해주세요"></input>
-            <input class="woowa-input login-form__pw" type="password" placeholder="비밀번호를 입력해주세요"></input>
-            <input class="woowa-btn login-form__submit-btn" type="submit" value="로그인"></input>
-        </form>
-        <button class="woowa-btn signup-btn" disabled>회원가입</button>
-    </main>
-    <script src="./assets/js/index.js"></script>
+  <header class="header-layout">
+      <div class="header-layout__title">My 배민</div>
+  </header>
+  <main class="main-page-layout">
+    <section class="main-section">
+      <div class="main-banner"></div>
+      <a class="login-btn">
+        <img class="login-btn__user" src="./assets/icons/user.png" alt="user_icon"/> 
+        <div class="login-btn__title">로그인 해주세요</div>
+        <img class="login-btn__icon" style="transform: rotate(180deg)" src="./assets/icons/left.svg" alt="right_icon"/>
+      </a>
+      <nav class="main-menu-layout">
+        <ul class="main-menu">
+          <a class="main-menu__item">
+            <img src="./assets/icons/point.png" alt="point_icon"/>
+            <span>포인트</span>
+          </a>
```

Comment:
ul태그를 사용하셨다면 li 태그와 함께 사용하는게 좋아보입니다...!!

```suggestion
        <ul class="main-menu">
          <li>
            <a class="main-menu__item">
              <img src="./assets/icons/point.png" alt="point_icon"/>
              <span>포인트</span>
            </a>
          </li>
```